src/seeders/20200904135720-countries.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
```

Comment:
in all your seeders, add all the necessary ids , you can get fake uuids using this link: https://www.uuidgenerator.net/version4

Reply:
done